docs/sources/staged.rst

```diff
@@ -196,6 +197,19 @@ Files in this directory must have metadata included in ``staged.yaml``.
 
 Will yield instances of :class:`~pushsource.AmiPushItem`.
 
+root/destination/CLOUD_IMAGES/\*
+..............................
+
+The ``CLOUD_IMAGES`` should contain the VMI(s) plus a ``resources.yaml``.
+
+The ``resources.yaml`` contains all the information needed for the
```

Comment:
The format of this will have to be documented somewhere.
Ideally it should have a schema, like the existing `errata-schema.yaml` and `staged-schema.yaml`, which could be linked from here.